core/ui/public/i18n/en/translation.json

```diff
@@ -1641,5 +1645,40 @@
         "available_with_subscription_title": "Activate subscription to enable this feature",
         "available_with_subscription": "Send a stream of security logs from applications and nodes of this cluster to Cloud Log Manager, where you can store, archive, and search them in a central place.",
         "cluster_id_notification": "Security logs from this cluster are available in the Cloud Log Manager UI under the host '{clusterId}'."
+    },
+    "applications": {
+        "title": "Applications",
+        "description": "Manage and install applications on your cluster",
+        "go_to_software_center": "Go to Software center",
+        "no_application": "No application installed",
+        "no_application_description": "Start by installing applications on this cluster",
+        "go_to_core_applications": "Core apps",
```

Comment:
We already have `software_center.core_apps`, let’s try to avoid the proliferation of unnecessary i18n strings
```suggestion
```